packages-next/keystone/src/artifacts.ts

```diff
@@ -105,20 +105,93 @@ export async function generateCommittedArtifacts(
   return artifacts;
 }
 
+const nodeAPIJS = (
+  cwd: string,
+  config: KeystoneConfig
+) => `import keystoneConfig from '../../keystone';
+import { PrismaClient } from '.prisma/client';
+import { createListsAPI } from '@keystone-next/keystone/___internal-do-not-use-will-break-in-patch/node-api';
+${makeVercelIncludeTheSQLiteDB(cwd, path.join(cwd, 'node_modules/.keystone/next'), config)}
+
+export const lists = createListsAPI(keystoneConfig, PrismaClient);
+`;
+
+const nodeAPIDTS = `import { KeystoneListsAPI } from '@keystone-next/types';
+import { KeystoneListsTypeInfo } from './types';
+
+export const lists: KeystoneListsAPI<KeystoneListsTypeInfo>;`;
+
+const makeVercelIncludeTheSQLiteDB = (
+  cwd: string,
+  directoryOfFileToBeWritten: string,
+  config: KeystoneConfig
+) => {
+  if (config.db.adapter === 'prisma_sqlite') {
+    const sqliteDbAbsolutePath = path.resolve(cwd, config.db.url.replace('file:', ''));
+
+    return `import path from 'path';
+
+    path.join(__dirname, ${JSON.stringify(
+      path.relative(directoryOfFileToBeWritten, sqliteDbAbsolutePath)
+    )});
+    path.join(process.cwd(), ${JSON.stringify(path.relative(cwd, sqliteDbAbsolutePath))});
+    `;
+  }
+  return '';
+};
+
+const nextGraphQLAPIJS = (
+  cwd: string,
+  config: KeystoneConfig
+) => `import keystoneConfig from '../../../keystone';
+import { PrismaClient } from '.prisma/client';
+import { nextGraphQLAPIRoute } from '@keystone-next/keystone/___internal-do-not-use-will-break-in-patch/next-graphql';
+${makeVercelIncludeTheSQLiteDB(cwd, path.join(cwd, 'node_modules/.keystone/next'), config)}
+
+export const config = {
+  api: {
+    bodyParser: false,
+  },
+};
+
+export default nextGraphQLAPIRoute(keystoneConfig, PrismaClient);
+`;
+
+// note the export default config is just a lazy way of going "this is also any"
+const nextGraphQLAPIDTS = `export const config: any;
+export default config;
+`;
+
 export async function generateNodeModulesArtifacts(
   graphQLSchema: GraphQLSchema,
   keystone: BaseKeystone,
+  config: KeystoneConfig,
```

Comment:
Note no changeset for this because there hasn't been a release since this was added.